libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -206,15 +300,36 @@ class AP_Airspeed
         Airspeed_Calibration calibration;
         float last_saved_ratio;
         uint8_t counter;
+
+        // Blending error (weight of sensor in blending)
+        float blending_factor;
     } state[AIRSPEED_MAX_SENSORS];
 
+    // Blended airspeed
+    AP_Int8 blend_conf;
```

Comment:
rename to blend.enable and move inside the struct

Reply:
OK